docs/commands.rst

```diff
@@ -0,0 +1,91 @@
+Management Commands
+===================
+
+
+Use Case
+--------
+These management commands are designed for use when there is a need to programmatically create
+or update Django users, groups, and permissions, without needing to hardcode these in migrations or
+application code.
+
+The commands are designed to work idempotently - that is, running them multiple times
+with the same inputs should always result in the same outcomes, and without additional side-effects if they
+are run more than once.
+
+
+manage_group
+------------
+The ``manage_group`` command is used to ensure that a specific group exists, and that it has a specific set of
```

Comment:
Minor: Link to the Group model docs at https://docs.djangoproject.com/en/1.8/ref/contrib/auth/#group-model.